stable/appmesh-controller/README.md

```diff
@@ -43,14 +45,92 @@ helm repo add eks https://aws.github.io/eks-charts
 Install the App Mesh CRDs:
 
 ```sh
-kubectl apply -k github.com/aws/eks-charts/stable/appmesh-controller//crds?ref=master
+kubectl apply -f https://raw.githubusercontent.com/aws/eks-charts/master/stable/appmesh-controller/crds/crds.yaml
```

Comment:
I would revert this change. Imagine someone bumps the CRD version to v1beta2 in appmesh-controller master and it takes some time to release a new controller image and a new chart version. In that period of time, all helm installs will be broken.